test/hpr_protocol_http_roaming_packet_SUITE.erl

```diff
@@ -0,0 +1,1588 @@
+%%%-------------------------------------------------------------------
+%%% @author jonathanruttenberg
+%%% @copyright (C) 2022, <COMPANY>
```

Comment:
Can we replace all the `<COMPANY>` by `Nova Labs`

```suggestion
%%% @copyright (C) 2022, Nova Labs
```

Reply:
Done.